### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -75,9 +63,11 @@ chown $app:www-data "$install_dir/include/ost-config.php"
 #=================================================
 ynh_script_progression --message="Building plugins..." --weight=1
 
-ynh_exec_as $app mkdir -p "$install_dir/include/plugins/build/.config/composer"
-export COMPOSER_HOME="$install_dir/include/plugins/build/.config/composer"
+mkdir -p "$install_dir/include/plugins/build/.config/composer"
+chmod 666 "$install_dir/include/plugins/build/.config/composer"
```
Comment (alexAubin) [inline scripts/install:67]:
```suggestion
chmod 660 "$install_dir/include/plugins/build/.config/composer"
```

:grimacing:

**Review by alexAubin — APPROVED** [review]

**Pull request merged**